[0.447s] Invoking command in '/home/akash/colcon_ws/src/build/ebot_docking': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble ftp_proxy=edcguest:edcguest@172.31.102.29:3128" /usr/bin/cmake /home/akash/colcon_ws/src/ebot_docking -DCMAKE_INSTALL_PREFIX=/home/akash/colcon_ws/src/install/ebot_docking
[0.519s] -- The C compiler identification is GNU 11.4.0
[0.590s] -- The CXX compiler identification is GNU 11.4.0
[0.599s] -- Detecting C compiler ABI info
[0.695s] -- Detecting C compiler ABI info - done
[0.708s] -- Check for working C compiler: /usr/bin/cc - skipped
[0.708s] -- Detecting C compile features
[0.709s] -- Detecting C compile features - done
[0.715s] -- Detecting CXX compiler ABI info
[0.829s] -- Detecting CXX compiler ABI info - done
[0.846s] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[0.846s] -- Detecting CXX compile features
[0.847s] -- Detecting CXX compile features - done
[0.856s] -- Found ament_cmake: 1.3.5 (/opt/ros/humble/share/ament_cmake/cmake)
[1.086s] Invoked command in '/home/akash/colcon_ws/src/build/ebot_docking' returned '-2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble ftp_proxy=edcguest:edcguest@172.31.102.29:3128" /usr/bin/cmake /home/akash/colcon_ws/src/ebot_docking -DCMAKE_INSTALL_PREFIX=/home/akash/colcon_ws/src/install/ebot_docking
[1.094s] Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 75, in execute
    loop.run_until_complete(future)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 603, in run_forever
    self._run_once()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 1871, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.10/selectors.py", line 469, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__
    rc = await self.task(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__
    return await task_method(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_cmake/build.py", line 59, in build
    rc = await extension.build(
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/build.py", line 87, in build
    rc = await self._reconfigure(args, env)
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/build.py", line 182, in _reconfigure
    buildfile = get_buildfile(cmake_cache)
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/__init__.py", line 112, in get_buildfile
    if 'Ninja' in generator:
TypeError: argument of type 'NoneType' is not iterable
